### ISSUE TITLE
Freeze versions of dependences

### ISSUE THREAD

**Issue by Krukov** [xhtml2pdf/xhtml2pdf#324]
I got error
```
root@70afd670efdc:/# pip freeze
html5lib==0.999999999
Pillow==3.3.0
PyPDF2==1.26.0
reportlab==3.3.0
six==1.10.0
virtualenv==14.0.6
webencodings==0.5
wheel==0.29.0
xhtml2pdf==0.0.6
You are using pip version 7.1.2, however version 8.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
root@70afd670efdc:/# python -c "from xhtml2pdf import pisa"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python2.7/site-packages/xhtml2pdf/pisa.py", line 3, in <module>
    from xhtml2pdf.document import pisaDocument
  File "/usr/local/lib/python2.7/site-packages/xhtml2pdf/document.py", line 2, in <module>
    from xhtml2pdf.context import pisaContext
  File "/usr/local/lib/python2.7/site-packages/xhtml2pdf/context.py", line 23, in <module>
    import xhtml2pdf.parser
  File "/usr/local/lib/python2.7/site-packages/xhtml2pdf/parser.py", line 17, in <module>
    from html5lib import treebuilders, inputstream
ImportError: cannot import name inputstream
root@70afd670efdc:/# 
```

I found out that `html5lib` have breaking changes at new version (https://github.com/html5lib/html5lib-python/issues/295),  but this could have been avoided if the version has been fixed.

**Comment by benjaoming** [comment]
https://github.com/xhtml2pdf/xhtml2pdf/pull/321

**Comment by benjaoming** [comment]
Okay thanks for reporting this, it's been fixed in 1.0b3
